File: JS/10-18-25/JS/Armstrong2.js
for (let number = 1; number <= 1000; number++){
	let num_string = String(number);
	let length = num_string.length;

	let sum = 0;

	for (let index = 0; index < length; index++){
		let char_digit = num_string.charAt(index);
		let digit  = Number(char_digit);
		sum += (digit ** length);

	if sum == number
		console.log(number);

File: JS/10-18-25/Python/Armstrong2.py
for number in range(1, 1001):
	num_string= str(number
	length = len(num_string)
	sum = 0

	for index in range(length):
		char_digit = num_string[index]
		digit = int(char_digit)
		sum += (digit ** length)

		if sum == number:
			print(number)